File: source/cpBase/CPUserErrorBaseClass.vb
'
' documentation should be in a new project that inherits these classes. The class names should be the object names in the actual cp project
'
Namespace Contensive.BaseClasses
    Public MustInherit Class CPUserErrorBaseClass
        'Public Sub New(ByVal cmcObj As Contensive.Core.cpCoreClass, ByRef CPParent As CPBaseClass)
        Public MustOverride Sub Add(ByVal Message As String) 'Implements BaseClasses.CPUserErrorBaseClass.Add
        Public MustOverride Function GetList() As String 'Implements BaseClasses.CPUserErrorBaseClass.GetList
        Public MustOverride Function OK() As Boolean 'Implements BaseClasses.CPUserErrorBaseClass.OK
    End Class

End Namespace

File: source/cpBaseCs/CPUserErrorBaseClass.cs
//========================================================================
// This conversion was produced by the Free Edition of
// Instant C# courtesy of Tangible Software Solutions.
// Order the Premium Edition at https://www.tangiblesoftwaresolutions.com
//========================================================================

//
// documentation should be in a new project that inherits these classes. The class names should be the object names in the actual cp project
//

//INSTANT C# NOTE: Formerly VB project-level imports:
using System;
using System.Collections;
using System.Collections.Generic;
using System.Diagnostics;

namespace Contensive.BaseClasses
{
	public abstract class CPUserErrorBaseClass
	{
		//Public Sub New(ByVal cmcObj As Contensive.Core.cpCoreClass, ByRef CPParent As CPBaseClass)
		public abstract void Add(string Message); //Implements BaseClasses.CPUserErrorBaseClass.Add
		public abstract string GetList(); //Implements BaseClasses.CPUserErrorBaseClass.GetList
		public abstract bool OK(); //Implements BaseClasses.CPUserErrorBaseClass.OK
	}

}

